akka-docs/src/main/paradox/typed/from-classic.md

```diff
@@ -0,0 +1,358 @@
+in Typed compared to classic.
+
+Akka Classic is the original Actor APIs, which have been improved by more type safe and guided Actor APIs,
+known as Akka Typed. Akka Classic is still fully supported and existing applications can continue to use
+the classic APIs. It is also possible to use Akka Typed together with classic actors within the same
+ActorSystem, see @ref[coexistence](coexisting.md). For new projects we recommend using the new Actor APIs.
+
+## Dependencies
+
+The dependencies of the Typed modules are named by adding `-typed` suffix of the corresponding classic
+module, with a few exceptions.
+
+For example `akka-cluster-typed`:
+
+@@dependency[sbt,Maven,Gradle] {
+  group=com.typesafe.akka
+  artifact=akka-cluster-typed_$scala.binary_version$
+  version=$akka.version$
+}
+
+Artifact names:
+
+| Classic               | Typed                       |
+|-----------------------|-----------------------------|
+| akka-actor            | akka-actor-typed            |
+| akka-cluster          | akka-cluster-typed          |
+| akka-cluster-sharding | akka-cluster-sharding-typed |
+| akka-cluster-tools    | akka-cluster-typed          |
+| akka-distributed-data | akka-cluster-typed          |
+| akka-persistence      | akka-persistence-typed      |
+| akka-stream           | akka-stream-typed           |
+| akka-testkit          | akka-actor-testkit-typed    |
+
+Cluster Singleton and Distributed Data are included in `akka-cluster-typed`.
+
+Artifacts not listed in above table don't have a specific API for Akka Typed.
+
+## Package names
+
+The convention of the package names in Akka Typed is to add `typed.scaladsl` and `typed.javadsl` to the
+corresponding Akka classic package name. `scaladsl` and `javadsl` is the convention to separate Scala and Java
+APIs, which is familiar from Akka Streams.
+
+Examples of a few package names:
+
+| Classic               | Typed for Scala                 | Typed for Java                 |
+|-----------------------|---------------------------------|--------------------------------|
+| akka.actor            | akka.actor.typed.scaladsl       | akka.actor.typed.javadsl       |
+| akka.cluster          | akka.cluster.typed              | akka.cluster.typed             |
+| akka.cluster.sharding | akka.cluster.sharding.scaladsl  | akka.cluster.sharding.javadsl  |
+| akka.persistence      | akka.persistence.typed.scaladsl | akka.persistence.typed.javadsl |
+
+## Actor definition
+
+A classic actor is defined by a class extending @scala[`akka.actor.Actor`]@java[`akka.actor.AbstractActor`].
+
+An actor in Typed is defined by a class extending @scala[`akka.actor.typed.scaladsl.AbstractBehavior`]@java[`akka.actor.typed.javadsl.AbstractBehavior`].
+
+It's also possible to define an actor in Typed from functions instead of extending a class. This is called
+the @ref:[functional style](style-guide.md#functional-versus-object-oriented-style).
+
+TODO snip
```

Comment:
I think it would be too ambitious to have side-by-side examples for all sections, but we can have a few for the most important.